crates/wasmparser/src/readers/component/canonicals.rs

```diff
@@ -269,9 +282,12 @@ impl<'a> FromReader<'a> for CanonicalFunction {
             0x04 => CanonicalFunction::ResourceRep {
                 resource: reader.read()?,
             },
-            0x05 => CanonicalFunction::ThreadSpawn {
+            0x05 => CanonicalFunction::ThreadSpawnRef {
                 func_ty_index: reader.read()?,
             },
+            0x24 => CanonicalFunction::ThreadSpawnIndirect {
+                table_index: reader.read()?,
+            },
```

Comment:
This makes sense logically to put here but for easily scanning this list mind moving it down to the bottom so it's more clear where the "holes" are in decoding?

Reply:
This is now resolved by moving to the `0x40+` space.